### PR TITLE
update rate limiting /1

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.learn.microsoft.com/en-us/aspnet/core/performance/rate-limit?view=aspnetcore-7.0&branch=pr-en-us-27294#enableratelimiting-and-disableratelimiting-attributes)

### CODE REVIEW

File: aspnetcore/performance/rate-limit.md [hunk 1]
```diff
@@ -127,6 +129,36 @@ The following code uses the concurrency limiter:
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs" id="snippet_concur":::
 
+## `EnableRateLimiting` and `DisableRateLimiting` attributes
```
Comment (Rick-Anderson) [inline aspnetcore/performance/rate-limit.md:132]: This section requires review and signoff.

File: aspnetcore/performance/rate-limit.md [hunk 2]
```diff
@@ -127,6 +129,36 @@ The following code uses the concurrency limiter:
...
+
+The [`[EnableRateLimiting]`](xref:Microsoft.AspNetCore.RateLimiting.EnableRateLimitingAttribute) and [`[DisableRateLimiting]`](xref:Microsoft.AspNetCore.RateLimiting.DisableRateLimitingAttribute) attributes can be applied to a Controller, action method, or Razor Page.
+
+The `[DisableRateLimiting]` ***disables*** rate limiting to the Controller, action method, or Razor Page regardless of other rate limiters applied. For example, consider the following code which calls <xref:Microsoft.AspNetCore.Builder.RateLimiterEndpointConventionBuilderExtensions.RequireRateLimiting%2A> to apply the `fixedPolicy` rate limiting to all controller endpoints:
```
Comment (wtgodbe) [inline aspnetcore/performance/rate-limit.md:136]:
```suggestion
The `[DisableRateLimiting]` attribute ***disables*** rate limiting to the Controller, action method, or Razor Page regardless of other rate limiters applied. For example, consider the following code which calls <xref:Microsoft.AspNetCore.Builder.RateLimiterEndpointConventionBuilderExtensions.RequireRateLimiting%2A> to apply the `fixedPolicy` rate limiting to all controller endpoints:
```

File: aspnetcore/performance/rate-limit.md (hunk 1 above)
Comment (wtgodbe) [reply aspnetcore/performance/rate-limit.md:132]: It's worth calling out that `DisableRateLimiting` disables both named policies _and_ the global limiter (if present)

File: aspnetcore/performance/rate-limit.md [hunk 3]
```diff
@@ -127,6 +129,36 @@ The following code uses the concurrency limiter:
...
+
+:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRate2/Program.cs" id="snippet_1" highlight="51":::
+
+In the following code, `[DisableRateLimiting]` disables rate limiting and overrides `[EnableRateLimiting("fixed")]` applied to the `Home2Controller` and `app.MapDefaultControllerRoute().RequireRateLimiting(fixedPolicy)` called in `Program.cs`:
+
+:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRate2/Controllers/Home2Controller.cs" id="snippet_1" highlight="1,16,22":::
+
+In the preceding code, the `[EnableRateLimiting("sliding")]` is ***not*** applied to the `Privacy` action method because `Program.cs` called `app.MapDefaultControllerRoute().RequireRateLimiting(fixedPolicy)`.
```
Comment (BrennanConroy) [inline aspnetcore/performance/rate-limit.md:144]: Is this true? I thought the inner most policy would apply.
@wtgodbe

**Pull request merged**